test/new.jl

```diff
@@ -2706,26 +2706,22 @@ for v in (nothing, "true")
     withenv("JULIA_PKG_USE_CLI_GIT" => v, "GIT_TERMINAL_PROMPT" => 0) do
         @testset "downloads with JULIA_PKG_USE_CLI_GIT = $v" begin
             isolate() do
-                @testset "libgit2 downloads" begin
-                    @testset "via name" begin
-                        Pkg.add(TEST_PKG.name; use_git_for_all_downloads=true)
+                @testset "via name" begin
+                    Pkg.add(TEST_PKG.name; use_git_for_all_downloads=true)
+                    @test haskey(Pkg.dependencies(), TEST_PKG.uuid)
+                    @eval import $(Symbol(TEST_PKG.name))
+                    @test_throws SystemError open(pathof(eval(Symbol(TEST_PKG.name))), "w") do io end  # check read-only
+                    Pkg.rm(TEST_PKG.name)
+                end
+                if (Base.get_bool_env("JULIA_PKG_USE_CLI_GIT", false) == false) && !Sys.iswindows()
+                    # TODO: fix. On GH windows runners cli git will prompt for credentials and hang.
```

Comment:
Should we do a `@test_skip` or something? Or a `@test ... skip=Sys.iswindows()`?

Reply:
Can we skip testsets like that? That's what would be needed here

Reply:
bump

Reply:
Oh, hmmm, skip the whole testset? I don't think that's possible. So we're probably stuck with the existing approach.